packages/compass-workspaces/src/components/workspaces.tsx

```diff
@@ -181,15 +185,17 @@ const CompassWorkspaces: React.FunctionComponent<CompassWorkspacesProps> = ({
             iconGlyph: 'Shell',
             tabTheme: getThemeOf(tab.connectionId),
           } as const;
-        case 'Databases':
+        case 'Databases': {
+          const connectionName = getConnectionTitleById(tab.connectionId);
           return {
             id: tab.id,
-            connectionName: getConnectionTitleById(tab.connectionId),
+            connectionName,
             type: tab.type,
-            title: tab.type,
+            title: multiConnectionsEnabled ? connectionName : tab.type,
             iconGlyph: 'Database',
```

Comment:
Ticket also mentions changing the icon here in addition to the title

Reply:
yes, this is work in progress, I was waiting to get some question around the icon answered